docs/guide/yaml/test/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Testing YAML Blueprints
+layout: website-normal
+children:
+- test-entities.md
+- usage-examples.md
+---
+
+Brooklyn provides a selection of basic test entities which can be used to validate Blueprints via YAML. These are divided into two groups; structural, which effect the order in which child entities are started; and validation, which are used to confirm the application is deployed as intended.
```

Comment:
Change "_structural, which effect the order in which child entities are started_" to "_structural,  which control the sequencing of service entity deployment and test entity execution_"